airflow/models.py

```diff
@@ -518,7 +519,7 @@ def is_superuser(self):
         return self.superuser
 
 
-class Connection(Base):
+class Connection(Base, LoggingMixin):
```

Comment:
Why this is needed?

Reply:
And how it relates with the change?

Reply:
@skudriashev this is needed because the class uses self.logger, which is provided by the mixin. Once the exception handling is fixed, you will have a subsequent error on calling self.logger https://github.com/apache/incubator-airflow/blob/17189ddcc8f7b8efaca6384ea854f5573e11ea4e/airflow/models.py#L610 if the mixin is not added.

Reply:
@GeorgeSirois ok, clear now. Thanks!